app/models/bulk_tagging_source.rb

```diff
@@ -0,0 +1,19 @@
+class BulkTaggingSource
+  def source_name_to_content_key_map
+    {
+      document_collection: 'documents',
+      topic: 'children',
+      mainstream_browse_page: 'children',
```

Comment:
It's because of the shape of the response from get_expanded_links.

Take a look at this in a console on your local or on integration: `Services.publishing_api.get_expanded_links("3e275a11-0fae-425b-a7a1-fe434594693f").to_h`